src/controllers/utils/queries.ts

```diff
@@ -3,36 +3,48 @@ import { cjkRange } from '../../shared/constants/diacriticCodes';
 import WordClass from '../../shared/constants/WordClass';
 import Tenses from '../../shared/constants/Tenses';
 import StopWords from '../../shared/constants/StopWords';
+import { Keyword } from './types';
+import { SearchRegExp } from '../../shared/utils/createRegExp';
+import { Filters } from '../types';
 
-const generateMultipleNsibidi = (keywords) => keywords.map(({ text }) => ({ 'definitions.nsibidi': text }));
+type Keywords = Partial<Keyword>[];
```

Comment:
again, we don't want to use `Partial` on `Keyword`

```suggestion
type Keywords = Keyword[];
```